src/test/java/com/google/step/datamanager/CommentManagerDatastoreTest.java

```diff
@@ -0,0 +1,106 @@
+package com.google.step.datamanager;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.step.model.Comment;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class CommentManagerDatastoreTest {
+
+  private static final long DEALID = 2;
+
+  private static final long USERID_A = 3;
+  private static final String CONTENT_A = "Hello world";
+
+  private static final long USERID_B = 4;
+  private static final String CONTENT_B = "Hello world2";
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig().setApplyAllHighRepJobPolicy());
+
+  private final CommentManagerDatastore commentManagerDatastore = new CommentManagerDatastore();
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void testCreateComment() {
+    Comment comment = commentManagerDatastore.createComment(DEALID, USERID_A, CONTENT_A);
+    assertEquals(DEALID, comment.dealId);
+    assertEquals(USERID_A, comment.userId);
+    assertEquals(CONTENT_A, comment.content);
+  }
+
+  @Test
+  public void testGetComments() {
+    Comment comment_A = commentManagerDatastore.createComment(DEALID, USERID_A, CONTENT_A);
```

Comment:
Always use `cameCase` for naming (non static) variables in java.
```suggestion
    Comment commentA = commentManagerDatastore.createComment(DEALID, USERID_A, CONTENT_A);
```